app/modules/edit_content.js

```diff
@@ -2,14 +2,30 @@ import { editable } from './selectors.js';
 
 export default function editTask() {
   const arrEdit = [...editable];
-  let editedText;
-  arrEdit.forEach((todo) => {
-    todo.addEventListener('keypress', (e) => {
-      if (e.key === 'Enter') {
-        e.preventDefault();
-        editedText = todo.innerText;
-      }
-      return editedText;
+
+  let tasks;
+
+  if (
+    localStorage.getItem('tasks') !== null
+    || localStorage.getItem('tasks') !== '[]'
+  ) {
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+  }
+
+  arrEdit.forEach((element) => {
+    const itemToEdit = element.innerText;
+    let editedText;
+    let editedArr;
+    element.addEventListener('input', () => {
```

Comment:
- [ ] The edited items are not getting persisted, kindly fix this issue 👍 
![Animation](https://user-images.githubusercontent.com/50721479/193342419-75f4691e-6645-4d11-b06d-6cdc98a53712.gif)